index.js

```diff
@@ -21,7 +21,7 @@ var extend = function (obj) {
   return obj;
 };
 
-var rcFinder = new RcFinder('.coffeelint.json', {
+var rcFinder = new RcFinder('coffeelint.json', {
```

Comment:
To avoid this being a breaking change, could you check for the existence of `coffeelint.json` first, and if it doesn't exist, fall back to `.coffeelint.json`? Otherwise this change will necessitate a major version change.